demos/python_demos/rag_chatbot/servable_stream/config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "model_config_list": [],
+    "mediapipe_config_list": [
+        {
+            "name":"python_model",
```

Comment:
hmm, maybe rename it to `rag`, so we get more specific?

Reply:
I think we should follow the convention from previous demos, all demos have exactly the same config.json and similar pbtxt